bin/get-localized-urls

```diff
@@ -36,7 +36,9 @@ https.get('https://resources.scratch.mit.edu/localized-urls.json', (res) => {
     res.on('end', () => {
         try {
             var urlJson = JSON.parse(urlData);
-            fs.writeFile(filename, JSON.stringify(urlJson, null, '  '));
+            fs.writeFile(filename, JSON.stringify(urlJson, null, '  '), (err) => {
+                if (err) process.stdout.write(`Failed writing file: ${err.message}\n`);
```

Comment:
The `catch` also exits the process with a non-zero exit code. Should this perform equivalently?